.github/workflows/k8s_tunnel.yml

```diff
@@ -0,0 +1,132 @@
+name: Tunnel to Juju controller via load balancer on k8s
+
+on:
+  pull_request:
+    paths-ignore:
+      - "README.md"
+      - "project-docs/**"
+  push:
+    branches:
+      - "main"
+    paths-ignore:
```

Comment:
Add `CHANGELOG.md` and the `examples` directory for both cases of pull_request and push. We should do that for all the workflows really.

Reply:
@cderici this one needs to be resolved please